backend/src/routers/gameRouter.ts

```diff
@@ -337,18 +337,71 @@ gameRouter.post('/remove', async (req, res) => {
  *    "message": "Game marked as played"
  *   }
  *
+ * @apiErrorExample {json} Unauthorized:
+ * HTTP/1.1 401 Unauthorized
+ *   {
+ *    "message": "Unauthorized"
+ *   }
+ *
+ * @apiErrorExample {json} Internal server Error:
+ * HTTP/1.1 500 Internal server error
+ *   {
+ *    "message": "Error marking game as not played"
+ *   }
  * @apiUse ZodError
  */
-gameRouter.post('/markPlayed', async (req, res) => {
+gameRouter.get('/markPlayed/:gameId', async (req, res) => {
```

Comment:
Even though you use a param, it should probably still be a `post` rather than a `get`, since it does mutate.